core/core-configuration-layer.el

```diff
@@ -184,8 +184,8 @@ subdirectory of ROOT is used."
             :initform 'unspecified
             :type (satisfies (lambda (x) (or (listp x) (eq 'unspecified x))))
             :documentation
-            (concat "A list of layers where this layer is enabled. "
-                    "(Takes precedence over `:disabled-for'.)"))
+            "A list of layers where this layer is enabled.
```

Comment:
you added an extra newline here

Reply:
good catch. I was over-zealous about keeping things under 80 columns wide